test/k8s-integration/main.go

```diff
@@ -74,6 +74,7 @@ const (
 	k8sInDockerBuildBinDir    = "_output/dockerized/bin/linux/amd64"
 	k8sOutOfDockerBuildBinDir = "_output/bin"
 	driverNamespace           = "gce-pd-csi-driver"
+	managedDriverNamespace    = "kube-system"
```

Comment:
can we make a function getDriverNamespace that does the *useGKEManagedDriver check? Otherwise someone will trip up in the future. And maybe change driverNamespace to externalDriverNamespace or something like that.

It looks like we only use it in a couple of places now; in installDriver we should keep using externalDriverNamespace explicitly.